### PR TITLE
[hcp|kv|disconnected] Various fixes for hcp|kv|disconnected periodics

### PR DESCRIPTION
**Pull request by tiraboschi**
Various fixes for hcp|kv|disconnected periodics:
- fix the right version of ODF for v4.14 and v41.5
- explicitly configure a ImageDigestMirrorSet for CSI sidecar images for ODF
- use MCE 2.6 on v4.16 job
- add a post step to eventually highlight missing images
- start reporting again v4.16 and v4.17 to slack
- execute with equinix-ocp-hcp cluster profile

### CODE REVIEW

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by YuLi517** [comment]
Account issue need to clarify. Open thread [here](https://redhat-internal.slack.com/archives/C058TF9K37Z/p1720680424389399).

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.15-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal-ovn-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected

**Comment by tiraboschi** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-metal-ovn-disconnected

**Comment by tiraboschi** [comment]
only 2/3 individual tests failed here and there, v4.14, v4.15 and v4.16 (with MCE) and v4.17 (without MCE) lanes worked e2e.
/pj-rehearse ack

File: ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ workflow:
     allow_best_effort_post_steps: true
     allow_skip_on_success: true
     post:
+    - ref: hypershift-kubevirt-baremetalds-disconnected-highlightmissingimages
```
Comment (LiangquanLi930) [inline ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml:26]: AFAIK, hypershift-mce-dump can accomplish the same thing.
E.g: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/54062/rehearse-54062-periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-kubevirt-metal-mce-disconnected/1812774648182476800/artifacts/e2e-kubevirt-metal-mce-disconnected/hypershift-mce-dump/artifacts/hostedcluster-6d047c994a7bcc9e751e/

Comment (tiraboschi) [reply ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml:26]: I'm also adding it to `hypershift-kubevirt-baremetalds-disconnected-conformance-workflow.yaml` which is not calling `hypershift-mce-dump`, but more than that is a matter of visibility: for sure we already have all the info we need to debug an issue in the must-gather archives we are already collecting, but getting at least one of the images not correctly mirrored on the internal registry is by far the most common issue in those disconnected lanes and this new step will `highlight` it if the issue is really there.

Comment (LiangquanLi930) [reply ci-operator/step-registry/hypershift/mce/kubevirt/baremetalds/disconnected/conformance/hypershift-mce-kubevirt-baremetalds-disconnected-conformance-workflow.yaml:26]: okay, Thanks

**Review by davidvossel — COMMENTED** [review]
/approve

**Pull request merged**